docs/build/tutorials/platform/adjusting-gas-price-during-high-network-activity.md

```diff
@@ -0,0 +1,110 @@
+---
+description: This tutorial will help users to adjust their priority fee and max fee cap during high network activity and take advantage of the benefits of dynamic fee transactions.
+---
+
+# Adjusting Gas Price During High Network Activity
+
+Sometimes during periods of high network activity, transactions either remain pending for a very long duration or instantly get a failed transaction notification. This may cause panic among users if they can't figure out what went wrong!
```

Comment:
Can we change 'This may cause panic among users' to something a bit less alarming, like 'This may confuse and frustrate users, especially if they don't understand why their transactions are not getting accepted.'

Reply:
Sounds good!